server/core/src/main/scala/org.apache.daffodil.debugger.dap/DAPodil.scala

```diff
@@ -63,6 +66,10 @@ object DAPSession {
 
       def sendEvent(event: DebugEvent): IO[Unit] =
         Logger[IO].info(show"<E $event") *> IO.blocking(server.sendEvent(event))
+
+      def terminate(message: String): IO[Unit] =
```

Comment:
@mbeckerle  So `terminate` is for stopping the debugger when an error has occurred. I can see why this should be renamed to abend or abort. Maybe should be moved somewhere instead of session since is causing the debugger to terminate or abort instead of the session?